android/src/main/kotlin/com/dns_technologies/mlkit_scanner/models/InitialScannerParameters.kt

```diff
@@ -0,0 +1,18 @@
+package com.dns_technologies.mlkit_scanner.models
+
+/** Describes initial parameters to initialize scanner  */
+data class InitialScannerParameters(
```

Comment:
`Initial` ИМХЛО лишнее. На использование это никаких ограничений не накладывает. 
```suggestion
data class ScannerParameters(
```
Для подобного смысла лучше именовать переменные или поля:
`val initialParams` и т.д.